src/main/java/gomedic/logic/parser/CliSyntax.java

```diff
@@ -9,6 +9,8 @@
     public static final String PREFIX_TYPE_DOCTOR = PREFIX_TYPE_AS_STRING + "doctor";
     public static final String PREFIX_TYPE_ACTIVITY = PREFIX_TYPE_AS_STRING + "activity";
     public static final String PREFIX_TYPE_PATIENT = PREFIX_TYPE_AS_STRING + "patient";
+    public static final String PREFIX_TYPE_ALL = PREFIX_TYPE_AS_STRING + "all";
```

Comment:
Instead of all, why dont the default `find` without the prefix all just search all the models